docs/platform/howto/set-authentication-policies.md

```diff
@@ -72,6 +72,13 @@ provider, then the token generated when the user was logged in with their passwo
 not work. After logging in with an allowed method on the new authentication policy
 the user can create a token.
 
+### Access from allowed IP addresses
+
+You can restrict users to accessing the Aiven Platform from specific IP address ranges
```

Comment:
It won't restrict the access to Aiven as a whole, but to the resources belonging to the organization that has IP allow list configured. For example,
User A exists in both org X and Y
if only org X has an IP allow list configured, the user will be required to have an IP in the configured range to access those resources(billing,project,servcie,support,etc..), however the user still has access to the platform and should be able to access resources under org Y for example.

Reply:
For managed users this will be different though. Managed users will be effectively locked out of Aiven unless the have an IP within the IP range specified in the allow list of the managing organization